manim/mobject/types/opengl_surface.py

```diff
@@ -0,0 +1,288 @@
+import numpy as np
+import moderngl
+
+from ...constants import *
+from ...mobject.opengl_mobject import OpenGLMobject
+from ...utils.bezier import integer_interpolate
+from ...utils.bezier import interpolate
+from ...utils.images import get_full_raster_image_path
+from ...utils.iterables import listify
+from ...utils.color import *
+from ...utils.space_ops import normalize_along_axis
+
+
+class OpenGLSurface(OpenGLMobject):
+    shader_dtype = [
+        ("point", np.float32, (3,)),
+        ("du_point", np.float32, (3,)),
+        ("dv_point", np.float32, (3,)),
+        ("color", np.float32, (4,)),
+    ]
+
```

Comment:
With this addition, my simple test scene

```py
class Test(Scene):
    def construct(self):
        s = OpenGLSphere()
        self.play(FadeIn(s))
        self.wait()
```

also produces some output.

```suggestion
    shader_folder = "surface"

```

Reply:
I added this change manually.